ietf/standard/management/commands/update_note_well.py

```diff
@@ -0,0 +1,38 @@
+from logging import Logger
+
+import markdown
+import requests
+from django.core.management.base import BaseCommand
+
+from ietf.standard.models import StandardPage
+
+logger = Logger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Update the note well page from the note well github repo"
+
+    def handle(self, *args, **options):
+        # Get content from github
+        note_well_url = (
+            "https://raw.githubusercontent.com/ietf/note-well/main/note-well.md"
```

Comment:
Can you move this out to the SETTINGS?
Also, it's better to use https://github.com/ietf/note-well/raw/main/note-well.md 
That URL redirects to the githubusercontent.com

Reply:
Done, and done.